p2p/peering.go

```diff
@@ -771,6 +773,8 @@ func (rp *RemotePeer) readMessages(ctx context.Context) error {
 				rp.receivedGetMiningState(ctx)
 			case *wire.MsgGetInitState:
 				rp.receivedGetInitState(ctx)
+			case *wire.MsgInitState:
+				rp.lp.receivedInitState <- newInMsg(rp, msg)
```

Comment:
there needs to be a mask for receiving this message. Otherwise, a caller that is not interest in this message will deadlock the local peer.  See the handling of *wire.MsgInv above for an example